### ISSUE TITLE
Confusion with BLE MAC

### ISSUE THREAD

**Issue by epenet** [raphmur/tesla-local-control-addon#20]
What should the format of the MAC be?

> BLE_MAC: Used for proximity discovery. Tesla's BLE MAC starts with "S" and ends with "C". To find the address, use "BLE scanner" on Android or "nRF Connect" on iOS.

Should it be `74:B8:39:XX:YY:ZZ` or should it be `SxxxxxxxxxxxxxxxxC`?
I ask because the documentation mentions `S...C`, but that seems to be auto-generated from the VIN, so there should be no need to ask it from the user.
If it should be the `74:B8...` code, then the documentation should be updated accordingly.

**Comment by iainbullock** [comment]
It's 74:B8:39:XX:YY:ZZ.

**Comment by baylanger** [comment]
@epenet just to make sure, is that 74:B8:39:... the WiFi MAC addr of your car? If so BLE_MAC != WiFi MAC addr

**Comment by epenet** [comment]
> @epenet just to make sure, is that 74:B8:39:... the WiFi MAC addr of your car? If so BLE_MAC != WiFi MAC addr

Yes I am sure.
Bluetooth scanner shows the name as S...C and the mac as 74:B8...

**Comment by epenet** [comment]
Note... something dodgy going on.
The car is currently 20km away from HA and its AddOn.
```
[19:55:49] INFO: Connecting to MQTT server; subscribe topics tesla_ble/+ and homeassistant/status
Reached 90 MQTT loops (~3min): Launch BLE scanning for car presence
Listening to BLE
74:B8:39:xx:yy:zz presence detected
```

**Comment by raphmur** [comment]
Hmmm... Are you sure of the time of the scan? Maybe the plugin crashed for another reason and did not update when car away.
I am facing this issue...

Regarding your first message, it should have been clarified in v0.0.7. Ultimately we may remove this parameter anyways as it can be automatically evaluated.

**Comment by raphmur** [comment]
#27

**Comment by baylanger** [comment]
> Note... something dodgy going on. The car is currently 20km away from HA and its AddOn.
> 
> ```
> [19:55:49] INFO: Connecting to MQTT server; subscribe topics tesla_ble/+ and homeassistant/status
> Reached 90 MQTT loops (~3min): Launch BLE scanning for car presence
> Listening to BLE
> 74:B8:39:xx:yy:zz presence detected
> ```
>
This probably happen due to #22 merged ~ 1hr ago.

**Comment by baylanger** [comment]
@epenet you can install 0.0.7a that has the above fix.

**Comment by raphmur** [comment]
Ok please keep posted.
Personally it works.
Close directly if solved

**Comment by epenet** [comment]
It's working fine in 0.0.8
I still think the translations should be modified though... If a MAC is expected call it a MAC, if it's the device name call it so, if both are acceptable say so :)

`Car's BLE MAC (aa:bb:cc:dd:ee:ff) or BLE Device name (S___________C); Use Android "BLE scanner" or iOS "nRF Connect"`
`Adresse MAC BLE (aa:bb:cc:dd:ee:ff) ou Nom BLE  (S___________C) du véhicule"`

**Comment by baylanger** [comment]
The fix for the name wasn't planned in 0.0.8 but in the next release.
